modules/ts/misc/trace_profiler.py

```diff
@@ -1,3 +1,26 @@
+#!/usr/bin/env python
+""" Parse OpenCV trace logs and present summarized statistics in a table
+
+### Options
+
+./trace_profiler.py <TraceLogFile> <num>
+
+<TraceLogFile>  - usually OpenCVTrace.txt
```

Comment:
Some instructions to get OpenCVTrace.txt are required or link to wiki, if it's not trivial.

Reply:
Added.